MdeModulePkg/Core/Dxe/DxeMain.h

```diff
@@ -14,6 +14,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #include <Protocol/LoadedImage.h>
 #include <Protocol/GuidedSectionExtraction.h>
+#include <Protocol/InternalEventServices.h>

```

Comment:
`MU_CHANGE` seems to be missing in a few places in the commit.

Reply:
Fixed the missing MU_CHANGE comment.